querygen/ujquery.go

```diff
@@ -98,6 +101,12 @@ var UJFieldSet = K8sAuditFieldSet{
 				)`,
 		},
 	},
+	K8sApiId{"appstudio.redhat.com","releases"}: {
+		"application": {
+			subObj: "properties",
+			srcExpr: `spath(_raw,"responseObject.metadata.ownerReferences{0}.name")`,
```

Comment:
Why do we need `spath`? This works and should perform better since the field is already indexed.

```
eval owner_name=mvindex('responseObject.metadata.ownerReferences{}.name', 0)
```

Reply:
done.